core/jvm/src/test/scala/scalaz/zio/RetrySpec.scala

```diff
@@ -119,29 +124,50 @@ class RetrySpec(implicit ee: org.specs2.concurrent.ExecutionEnv) extends TestRun
   }
 
   def retryN = {
-    val retried  = unsafeRun(retryCollect(IO.fail("Error"), Schedule.recurs(5)))
-    val expected = (Left("Error"), List(1, 2, 3, 4, 5, 6).map((Duration.Zero, _)))
+    val retried = unsafeRun(retryCollect(IO.fail("Error"), Schedule.recurs(5)))
+    val expected =
+      (Left("Error"), List(1, 2, 3, 4, 5, 6).map((Delay.none, _)))
     retried must_=== expected
   }
 
   def retryNUnitIntervalJittered = {
-    val schedule: ZSchedule[Random, Int, Int] = Schedule.recurs(5).delayed(_ => 500.millis).jittered
-    val scheduled: List[(Duration, Int)] = unsafeRun(
+    val schedule: ZSchedule[Random, Int, Int] =
+      Schedule.recurs(5).delayed(_ => 500.millis.relative).jittered
+    val scheduled: List[(Delay, Int)] = unsafeRun(
```

Comment:
Push this unsafe run to the top, by making a top-level for comprehension. Each test should have just 1 unsafeRun.